cypress/support/step_definitions/testSetup.js

```diff
@@ -51,6 +51,9 @@ Given('the environment has been set up for {string} tests', (test) => {
     cy.getCapabilities();
     destroyAppInstance(test);
     Cypress.env(CONSTANTS.ENV_SETUP_STATUS, true);
+    if (Cypress.env(CONSTANTS.TEST_TYPE).includes('rpc-Only')) {
```

Comment:
If we have this here do we need the one in the glue fireboltCall GC as well?

Reply:
removed from fireboltCalls.js